telegraf/rootfs/config.toml.tpl

```diff
@@ -204,6 +204,7 @@
 {{ if .PROMETHEUS_URLS }}
 [[inputs.prometheus]]
   urls = [{{ .PROMETHEUS_URLS }}]
+  insecure_skip_verify = {{ default true .PROMETHEUS_INSECURE_SKIP_VERIFY }}
```

Comment:
https://github.com/deis/controller/blob/master/rootfs/scheduler/__init__.py#L340-L345

We do the same for the controller but are looking at fixing that before 2.0